src/main/java/teamrtg/highlandsut/block/HighlandsBlocks.java

```diff
@@ -96,17 +96,17 @@ public static void constructBlocks() {
             OreDictionary.registerOre("treeLeaves", leaves[i]);
             OreDictionary.registerOre("treeSapling", saplings[i]);
 
-            Blocks.fire.setFireInfo(leaves[i], 30, 60);
-            Blocks.fire.setFireInfo(planks[i], 5, 20);
-            Blocks.fire.setFireInfo(woods[i], 5, 5);
+            Blocks.FIRE.setFireInfo(leaves[i], 30, 60);
+            Blocks.FIRE.setFireInfo(planks[i], 5, 20);
+            Blocks.FIRE.setFireInfo(woods[i], 5, 5);
         }
 
         for (int i = 0; i < NUM_PLANTS; i++) {
             plants[i] = new BlockHighlandsPlant(EnumTypePlant.META_LOOKUP[i].name);
 
             GameRegistry.registerBlock(plants[i], plants[i].getUnlocalizedName().substring(15));
```

Comment:
`GameRegistry#registerBlock(Block block, String name)` is deprecated
need to update to `GameRegistry#register(Block block, ResourceLocation location)`

So, instead of:
```java
            GameRegistry.registerBlock(
                plants[i],
                plants[i].getUnlocalizedName().substring(15)
            );
```
it will be something like:
```java
            GameRegistry.register(
                    plants[i].getBlockState().getBlock(),
                    new ResourceLocation(plants[i].getBlockState().getBlock().getUnlocalizedName())
            );
```